### PR TITLE
Branch fix issues

### PR DESCRIPTION
**Pull request by qinguorui2001**

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -69,6 +69,7 @@ public void resetData(ReadOnlyAddressBook newData) {
         requireNonNull(newData);
 
         setPersons(newData.getPersonList());
+        //activePerson = newData.getPersonList().get(0);
```
Comment (TaoJun99) [inline src/main/java/seedu/address/model/AddressBook.java:72]: Is this line for future use or is this supposed to be removed?

File: src/main/java/seedu/address/model/assignment/DueDate.java [hunk 2]
```diff
@@ -27,7 +28,8 @@
 
     protected static final DateTimeFormatter PARSE_DATE_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy");
     protected static final DateTimeFormatter PARSE_TIME_FORMAT = DateTimeFormatter.ofPattern("HHmm");
-    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a");
+    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a",
+            Locale.ENGLISH);
```
Comment (TaoJun99) [inline src/main/java/seedu/address/model/assignment/DueDate.java:32]: Not sure about the "Locale.ENGLISH" part, but is this part supposed to be included for us as well?

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -143,4 +152,15 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredPersonList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the person's assignment at the given {@code targetIndex}
+     * in the {@code model}'s address book.
+     */
+    public static void showAssignmentAtIndex(Model model, Person person, Index targetIndex) {
+        //Assignment assignment = TypicalAssignments.getTypicalAssignments().get(targetIndex.getZeroBased());
+        // person.getAssignments().add(assignment);
```
Comment (TaoJun99) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:161]: Same as first comment

File: src/test/java/seedu/address/logic/commands/DeleteAssignmentCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+        // ensures that outOfBoundIndex is still in bounds of address book list
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getAssignmentsList().size());
+
+        DeleteAssignmentCommand deleteAssignmentCommand =
+                new DeleteAssignmentCommand(new PersonBuilder().build().getName(), outOfBoundIndex);
+
+        assertCommandFailure(deleteAssignmentCommand, model, Messages.MESSAGE_INVALID_ASSIGNMENT_DISPLAYED_INDEX);
+    }
+
+    */
```
Comment (TaoJun99) [inline src/test/java/seedu/address/logic/commands/DeleteAssignmentCommandTest.java:179]: Same as first comment

**Review by TaoJun99 — COMMENTED** [review]
Test cases look good!

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 3 above)
Comment (qinguorui2001) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:161]: this is supposed to be modified when i submit the delete and mark assignment test, should be included now

File: src/test/java/seedu/address/logic/commands/DeleteAssignmentCommandTest.java (hunk 4 above)
Comment (qinguorui2001) [reply src/test/java/seedu/address/logic/commands/DeleteAssignmentCommandTest.java:179]: for further modification, should be included

Comment (TaoJun99) [reply src/test/java/seedu/address/logic/commands/DeleteAssignmentCommandTest.java:179]: Alright noted!

**Pull request merged**